src/Command.js

```diff
@@ -90,7 +90,7 @@ class Command {
       .then(msg => {
         this.self.counts.msgsSent = this.self.counts.msgsSent + 1
         if (deleteDelay) {
-          if (!this.deleteAfter || !this.config.deleteCommandMessages) return resolve(msg)
+          if (this.deleteAfter) return resolve(msg)
```

Comment:
Hey @washingdone. If this is tested and you confirmed it works, can you apply this fix to the rest of the code lines where `if (!this.deleteAfter || !this.config.deleteCommandMessages) return resolve(msg)` appears?
If I am not wrong the only places where that happens are [here](https://github.com/TheRacingLion/Discord-SelfBot/blob/master/src/Command.js#L73), [here](https://github.com/TheRacingLion/Discord-SelfBot/blob/master/src/Command.js#L93) and [here](https://github.com/TheRacingLion/Discord-SelfBot/blob/master/src/Command.js#L115). (The first one behind the line that is fixed in here)
Would be a great help.